### ISSUE TITLE
Swipe button in right side automatically opened and closed when enters to screen every time

### ISSUE THREAD

**Issue by Sakthimalar** [jemise111/react-native-swipe-list-view#264]

**Comment by olejka91** [comment]
have same issue

**Comment by jemise111** [comment]
@Sakthimalar @olejka91 Do you guys have some example code or a gif of the problem. I'm not sure I'm understanding what's going on? Thanks

**Comment by pbfrias17** [comment]
Also running into this issue. Here's a (gigantic) gif of it happening. As you can see, as soon as the page refreshes the rows swipe to the left.


![swipe-list](https://user-images.githubusercontent.com/5976642/51224785-8ea9a180-1915-11e9-935a-c2aaa360fb0f.gif)

**Comment by pbfrias17** [comment]
Here's the component code
```
<SwipeListView
  useFlatList
  disableRightSwipe
  data={addresses}
  renderItem={data => <AddressRow address={data.item} />}
  renderHiddenItem={() => (
     <TouchableOpacity
         onPress={() => this.props.removeAddress()}
         style={{ backgroundColor: 'pink', flex: 1 }}
     />
  )}
  rightOpenValue={-100}
/>
```

**Comment by Sakthimalar** [comment]
Guys Thanks for the response. My problem is solved by adding
`previewOpenValue={0.01}`

**Comment by pbfrias17** [comment]
Yup. I did not read documentation very well. That being said, is this still a bug? I don't think using `leftOpenValue` results in the same behavior

**Comment by jemise111** [comment]
@pbfrias17 @Sakthimalar Glad the issue is solved but just wanted to double check. Is there an instance in which you're seeing row's slide open automatically _without_ passing `previewRowKey`? If so that's def a bug
